client/src/App.jsx

```diff
@@ -1,28 +1,6 @@
-import logo from "./logo.svg";
-import "./App.css";
+import Navbar from "./Navbar/Navbar";
```

Comment:
It would be best to refactor the location of the navbar to a `components` directory. This would be similar to the linter example that I sent you way in this branch: [https://github.com/uoftblueprint/tces/tree/granthamblin0/tces-9-urd-navbar-LINTER-EXAMPLES](https://github.com/uoftblueprint/tces/tree/granthamblin0/tces-9-urd-navbar-LINTER-EXAMPLES)